package.json

```diff
@@ -15,6 +15,7 @@
     "plantuml-encoder": "^1.4.0",
     "prop-types": "^15.8.1",
     "react": "^17.0.2",
+    "react-csv": "^2.2.2",
```

Comment:
I need this dependency for the large CSV file download. The `a` tag method only works for 10M files.